website/www/site/layouts/partials/header.html

```diff
@@ -81,6 +81,15 @@
        <li>
          <a class="navbar-link" href={{ "/case-studies/" | relLangURL }}>{{ T "nav-case-studies" }}</a>
        </li>
+        <li>
+          <span class="navbar-link">Performance</span>
```

Comment:
we may put the dashboards under documentation/io/connectors at this moment; we may or may not need this as an extra top level tab on Tab when this is polished

Reply:
what do you think @pranavbhandari24 @liferoad

Reply:
@damondouglas could we move this to `documentation/io/connectors`? I do think later we can move this back to the top level once they are polished.

Reply:
If this is non-trivial we can just comment out the top level navbar at this moment (and add a entry link in io documentation)

Reply:
Per our discussion in chat I will resolve this.